### ISSUE TITLE
 db upgrade importError:No module named 'contextlib2'

### ISSUE THREAD

**Issue by zhangjian0111** [apache/incubator-superset#5004]
Make sure these boxes are checked before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if any
- [ ] I have reproduced the issue with at least the latest released version of superset
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar

**Comment by mistercrunch** [comment]
Which version of python? Which os/version? Please provide the full stacktrace.

**Comment by veszig** [comment]
I have the same issue. After upgrading I get the following error message when I try to upgrade the db:

```
superset db upgrade                                             
Traceback (most recent call last):
  File "/mnt/srv/superset/venv/bin/superset", line 12, in <module>
    from superset.cli import manager
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/__init__.py", line 179, in <module>
    ConnectorRegistry.register_sources(module_datasource_map)
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/connectors/connector_registry.py", line 20, in register_sources
    module_obj = __import__(module_name, fromlist=class_names)
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/connectors/sqla/__init__.py", line 3, in <module>
    from . import views  # noqa
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/connectors/sqla/views.py", line 19, in <module>
    from superset.connectors.base.views import DatasourceModelView
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/connectors/base/views.py", line 11, in <module>
    from superset.views.base import SupersetModelView
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/views/__init__.py", line 3, in <module>
    from . import core  # noqa
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/views/core.py", line 36, in <module>
    from superset import (
  File "/mnt/srv/superset/venv/local/lib/python2.7/site-packages/superset/sql_lab.py", line 12, in <module>
    from contextlib2 import contextmanager
ImportError: No module named contextlib2
```
Versions:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.3 LTS
Release:        16.04
Codename:       xenial
$ python -V
Python 2.7.12
```

**Comment by zhangjian0111** [comment]
it is fixed by run pip install contextlib2.
my version :
Distributor ID: Ubuntu
Description:    Ubuntu 16.04.3 LTS
Python 3.5.2

**Comment by mistercrunch** [comment]
Fixed in `0.25.5` release
